spec.md

```diff
@@ -31,9 +31,21 @@ sequence of operations the program encapsulates.
 - `"format"`: Signifies the utilization of the PHIR/JSON format.
 - `"version"`: Represents the semantic version number adhering to the PHIR spec.
 - `"metadata"`: An optional segment where users can incorporate additional details. This segment holds potential for
-future expansion, possibly to guid compilation processes and error modeling.
+future expansion, possibly to guide compilation processes and error modeling.
 - `"ops": [{...}, ...]`: A linear sequence denoting the operations and blocks that constitute the program.
 
+## Metadata
+
+The metadata field takes several parameters that can be used to specify information relevant to the entire program
+
+### Metadata Options
+
+| parameter               | options                 | description                                                     |
+| ----------------------- | ----------------------- | --------------------------------------------------------------- |
+| `"default_time_unit"`   | `"s", "ms", "us", "ns"` | The default time unit in the "duration" section of qop metadata |
+| `"default_angle_unit"`  | `"rad", "pi"`           | The default angle unit in the "angles" section of a qop         |
+| `"strict_parallelism"`  | `"true", "false"`       | If true, interperet qops with multiple qubits outside a parallel block as the same gate being executed in parallel on all qubits. If false, interperet qops with multiple qubits outside a parallel block as the same gate being executed serially on each qubit in turn. | {{< mdl-disable "<!-- markdownlint-disable MD013 -->" >}}
```

Comment:
```suggestion
| `"strict_parallelism"`  | `"true", "false"`       | If `"true"`, interpret `qop`s with multiple arguments as parallel application of the `qop` to those arguments. If `"false"` (the default), interpret qops with multiple qubits outside a parallel block as the same gate being executed serially on each qubit in turn. | {{< mdl-disable "<!-- markdownlint-disable MD013 -->" >}}
```

Didn't we say that `false` implies that the simulator gets to choose? It doesn't mean strictly serial, right?